dashboard/test/controllers/plc/evaluation_questions_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class Plc::EvaluationQuestionsControllerTest < ActionController::TestCase
+  setup do
+    user = create (:admin)
```

Comment:
Nit: extra space after `create` changes the meaning.  It has the same result with one argument, but adding a second argument would throw an exception.